### PR TITLE
Updates from rspec-dev (2020-09-28)

### PR DESCRIPTION
**Pull request by benoittgt**
These are some updates, generated from rspec-dev's rake tasks.

https://github.com/rspec/rspec-dev/pull/250

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,15 +39,12 @@ matrix:
...
       env: JRUBY_OPTS='--dev --1.8'
-    - rvm: 2.7.1
-      env: DIFF_LCS_VERSION="~> 1.3.0"
   allow_failures:
     - rvm: jruby-head
     - rvm: ruby-head
-    - rvm: ruby-3.0.0-preview1
     - rvm: rbx-3
   fast_finish: true
 branches:
```
Comment (pirj) [inline .travis.yml:47]: Wondering if this is correct at all. What is this for? I see both `pr` and `push` builds. Is this supposed to trigger a `push` build only for certain branches? I couldn't find this in the doc.
There's this, but it's still not clear how to exclusively run the `pr` build https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

File: .travis.yml [hunk 2]
```diff
@@ -39,15 +39,12 @@ matrix:
   include:
     - rvm: jruby-1.7
       env: JRUBY_OPTS='--dev --1.8'
-    - rvm: 2.7.1
-      env: DIFF_LCS_VERSION="~> 1.3.0"
```
Comment (JonRowe) [inline .travis.yml:43]: Needs restoring

File: .travis.yml (hunk 1 above)
Comment (JonRowe) [reply .travis.yml:47]: PR's always run, builds can also run on pushes to branches.

File: .travis.yml [hunk 3]
```diff
@@ -39,15 +39,12 @@ matrix:
...
   allow_failures:
     - rvm: jruby-head
     - rvm: ruby-head
-    - rvm: ruby-3.0.0-preview1
     - rvm: rbx-3
   fast_finish: true
 branches:
   only:
-    - main
+    - master
```
Comment (JonRowe) [inline .travis.yml:49]: Should be main :)

**Review by JonRowe — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (benoittgt) [reply .travis.yml]: arg my bad

**Review by JonRowe — APPROVED** [review]

**Pull request merged**